src/v0/destinations/freshsales/transform.js

```diff
@@ -128,6 +128,7 @@ function eventMappingHandler(message, destination) {
   if (!event) {
     throw new InstrumentationError('Event name is required');
   }
+  validateEventName(event);
```

Comment:
can you check if adding this validation just here - https://github.com/rudderlabs/rudder-transformer/pull/2795/files#diff-b8b4d74006ce5873ee454743fb5c00c3ac2a1d37dbbfb41567d9207909a53e8fR164 should suffice ? And remove it from eventMappingHandler & trackResponseBuilder

Reply:
Added the validation one level up and removed it from eventMappingHandler & trackResponseBuilder